topgg-page.htm

```diff
@@ -4,75 +4,174 @@
+            /* lists stuff */
             li {
                 color:white;
                 font-family: Arial, Helvetica, sans-serif;
                 text-align: left;
+                margin-left: 30px;
+            }
+            #bubble-list {
+                background-image: linear-gradient(90deg, #1e143b8a, rgba(0, 0, 0, .5), transparent);
+                border-radius: 20px;
+                border: 2px solid black;
+                padding: 10px;
+            }
+            .dropdown {
+                position: relative;
+                display: inline-block;
+            }
+            .dropdown-center {
+                text-align: center;
+            }
+            .dropdown-content {
+                display: none;
+                position: absolute;
+                margin-right: 50px;
+                margin-left: 50px;
+                width: calc(100vw/3);
+                height: auto;
+                background-color: var(--main-color);
+                box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+                padding: 12px 16px;
+                z-index: 1;
+            }
+
+            .dropdown:hover .dropdown-content {
+                display: block;
+                margin-bottom: 30px;
+            }
+            /* buttons */
+            .main-links {
+                text-align: center;
+            }
+            input.button {
+                -webkit-appearance: button;
+                -moz-appearance: button;
+                appearance: button;
+                color: white;
+                background-color: #0e0e0e;
+                border-radius: 5px;
+                border-color: transparent;
+                font-family: kalam;
+            }
+            input.button:hover {
+                background-color: black
             }
         </style>
     </head>
     <body>
         <h1>Radeon Development</h1>
+        <div class="main-links">
+            <input class="button" type="button"  onclick="location.href='https://discord.gg/xcZwGhSy4G';" value="Support Server" />
+            <input class="button" type="button" onclick="location.href='https://discord.com/api/oauth2/authorize?client_id=762359941121048616&permissions=8&scope=bot';" value="Invite Radeon" />
+            <input class="button" type="button" onclick="location.href='https://github.com/devnote-dev/Radeon';" value="Github Repo" />
+        </div>
         <p>
-            Radeon is a multi-functional moderation bot with an anti-raid system! While still a small bot, it has a powerful and up-to-date moderation system with active development on the anti-raid system. Any and all support is appreciated!
+            Radeon is a multi-functional moderation bot with an anti-raid system!<br>
+            While still a small bot, it has a powerful and up-to-date moderation system with active development on the anti-raid system.<br>
+            Any and all support is appreciated!
         </p>
+
+        <br><br><br><br>
```

Comment:
this is actually bad practise and abuse of the `<br>` tag ([find out why](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element)), there are far better ways such as using CSS:
HTML used:
```html
<p id="spacer">
</p>
```
using the [box model](https://www.w3schools.com/Css/css_boxmodel.asp), the CSS would look something like this:
```css
#spacer {
margin-top:  3em;
margin-bottom: 3em;
}
```
might need some adjusting regarding spacing, but gets the point across
em is a magical unit, [more on units in css](https://www.w3.org/Style/Examples/007/units.en.html).